beakerx/js/src/plot/plotScope.js

```diff
@@ -2417,7 +2417,7 @@ define([
       self.model.model = data;
     }
 
-    // self.element = $('div#'+self.wrapperId+' .dtcontainer');
+    self.element = $('div#'+self.wrapperId+' .dtcontainer');
```

Comment:
we can put it inside if statement to be sure that it only assigns when there is treemap